central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl.go

```diff
@@ -60,23 +60,29 @@ func (ds *datastoreImpl) ScanConfigurationExists(ctx context.Context, scanName s
 
 // GetScanConfigurations retrieves the scan configurations specified by query
 func (ds *datastoreImpl) GetScanConfigurations(ctx context.Context, query *v1.Query) ([]*storage.ComplianceOperatorScanConfigurationV2, error) {
-	if ok, err := scanConfigurationsSAC.ReadAllowed(ctx); err != nil {
-		return nil, err
-	} else if !ok {
-		return nil, nil
-	}
-
-	return ds.storage.GetByQuery(ctx, query)
+	scanConfigs, err := ds.storage.GetByQuery(ctx, query)
+
+	// SAC will return a config if a user has permissions to ANY of the clusters.  For tech preview, and
```

Comment:
Please read this comment to make sure I'm making sense.  The model here is a ScanConfig object applies to multiple clusters.  I do not repeat the details of the ScanConfig object because it needs to remain in sync.

Reply:
Do other stackrox objects have this issue, too? I dug around in the policy UI, but didn't find a way to associate a policy to multiple clusters.

Reply:
Policies are global scoped in terms of SAC, violations belong to a single cluster, as do deployments. The only thing perhaps is Image?